pages/interface/utils/trusted-domain.js

```diff
@@ -0,0 +1,12 @@
+const TRUSTED_HOSTS = ['curso.dev', 'github.com', 'tabnews.com.br'];
+
+export default function isTrustedDomain(url) {
+  const urlWithProtocol = url.startsWith('http') ? url : `https://${url}`;
+  const { hostname } = new URL(urlWithProtocol);
```

Comment:
Na implementação atual, a `url` nunca irá chegar aqui sem protocolo. No mínimo vem com o protocolo relativo `//`.

Mas realmente é bom deixar a função pronta para os outros casos, e ela não iria quebrar com uma `url` com protocolo relativo, como `//tabnews.com.br`, mas a `urlWithProtocol` ficaria bem estranha: `https:////tabnews.com.br`.

Então eu acho que podemos fazer algo como a sugestão abaixo, pois onde quer que a função seja utilizada, ficará coerente com o comportamento do ByteMD, que irá sempre criar um link relativo se não for fornecido o protocolo.

```suggestion
  const { hostname } = new URL(url, webserver.host);
```

Faz sentido?

Obs. Não vejo problemas em ter o `tabnews.com.br` no `TRUSTED_HOSTS`, e usar também o `webserver.host`, pois nem sempre terão o mesmo hostname.

Reply:
Tudo bem, podemos deixar o mesmo tipo de comportamento do ByteMD e aí deixo isso claro por meio de testes :+1: